lib/cluster.rb

```diff
@@ -120,6 +120,38 @@ def format
     @format ||= CalculateFormat.new(self).cluster_format
   end
 
+  def organizations_in_cluster
+    @organizations_in_cluster ||= (holdings.pluck(:organization) +
+                                  ht_items.pluck(:billing_entity)).uniq
+  end
+
+  # Maps enumchrons to list of orgs that have an item with that enumchron
+  # Not necessary for SPMs/SERs
+  def item_enum_chron_orgs
+    return @item_enum_chron_orgs if @item_enum_chron_orgs
+
+    @item_enum_chron_orgs = Hash.new {|h, k| h[k] = [] }
+    ht_items.each {|ht| @item_enum_chron_orgs[ht.n_enum] << ht.billing_entity }
+    @item_enum_chron_orgs
```

Comment:
Could write as:

```ruby
@item_enum_chron_orgs ||= ht_items.group_by { |ht| ht.n_enum }
  .transform_values { |htitems| htitems.map(&:billing_entity) }
```

I bet @billdueber  might even have a more concise way to do this...